<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreditCalculator</class>
 <widget class="QMainWindow" name="CreditCalculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>592</width>
    <height>440</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Times New Roman</family>
   </font>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Кредитный калькулятор</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: gray;
 ; color: rgb(254, 255, 254); border: none;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="creditCalcButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>370</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
     background-color: rgb(246, 153, 6); color: rgb(245, 246, 246); border-radius: 15px;  margin: 4px;
   
    }
QPushButton:pressed {  
    background-color: rgb(245, 246, 246);
	color: rgb(246, 153, 6);
 }

</string>
    </property>
    <property name="text">
     <string>Расчитать</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cleanCreditButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>370</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
     background-color: rgb(165, 165, 165); color: rgb(0, 0, 0); border-radius: 15px;  margin: 4px;
   
    }
QPushButton:pressed {  
    background-color: rgb(0, 0, 0);
	color: rgb(165, 165, 165);
 }

</string>
    </property>
    <property name="text">
     <string>Очистить</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelHistory_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>0</y>
      <width>371</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: gray;
 ; color: rgb(254, 255, 254); border: none;</string>
    </property>
    <property name="text">
     <string>Кредитный калькулятор</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelHistory_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: gray;
 ; color: rgb(254, 255, 254); border: none;</string>
    </property>
    <property name="text">
     <string>Сумма кредита</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelHistory_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: gray;
 ; color: rgb(254, 255, 254); border: none;</string>
    </property>
    <property name="text">
     <string>Срок (в месяцах)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelHistory_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>140</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: gray;
 ; color: rgb(254, 255, 254); border: none;</string>
    </property>
    <property name="text">
     <string>Процентная ставка (%)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="sumSpinBox">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>50</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: rgb(254, 255, 254); color: rgb(0, 0, 0); border: none;</string>
    </property>
    <property name="decimals">
     <number>2</number>
    </property>
    <property name="minimum">
     <double>0.000000000000000</double>
    </property>
    <property name="maximum">
     <double>100000000.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>100000.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="termSpinBox">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>100</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: rgb(254, 255, 254); color: rgb(0, 0, 0); border: none;</string>
    </property>
    <property name="decimals">
     <number>0</number>
    </property>
    <property name="minimum">
     <double>0.000000000000000</double>
    </property>
    <property name="maximum">
     <double>720.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>12.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="percentSpinBox">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>140</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: rgb(254, 255, 254); color: rgb(0, 0, 0); border: none;</string>
    </property>
    <property name="decimals">
     <number>2</number>
    </property>
    <property name="minimum">
     <double>0.000000000000000</double>
    </property>
    <property name="maximum">
     <double>100.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="labelHistory_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: gray;
 ; color: rgb(254, 255, 254); border: none;</string>
    </property>
    <property name="text">
     <string>Переплата по кредиту</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelHistory_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: gray;
 ; color: rgb(254, 255, 254); border: none;</string>
    </property>
    <property name="text">
     <string>Общая выплата</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelHistory_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: gray;
 ; color: rgb(254, 255, 254); border: none;</string>
    </property>
    <property name="text">
     <string>Ежемесячный платеж</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="monthlyPayLineEdit">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>210</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: rgb(254, 255, 254); color: rgb(0, 0, 0); border: none;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="maxLength">
     <number>255</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="cursorMoveStyle">
     <enum>Qt::LogicalMoveStyle</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="overpaymentLineEdit">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>260</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: rgb(254, 255, 254); color: rgb(0, 0, 0); border: none;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="maxLength">
     <number>255</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="cursorMoveStyle">
     <enum>Qt::LogicalMoveStyle</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="paymentLineEdit">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>310</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: rgb(254, 255, 254); color: rgb(0, 0, 0); border: none;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="maxLength">
     <number>255</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="cursorMoveStyle">
     <enum>Qt::LogicalMoveStyle</enum>
    </property>
   </widget>
   <widget class="QRadioButton" name="autonomousRadioButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>50</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: gray;
 ; color: rgb(254, 255, 254); border: none;</string>
    </property>
    <property name="text">
     <string>Аннуитентный</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="differentiatedRadioButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>90</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: gray;
 ; color: rgb(254, 255, 254); border: none;</string>
    </property>
    <property name="text">
     <string>Дифференцированный</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
